bootstrapper/internal/helm/helm.go

```diff
@@ -57,8 +57,10 @@ func New(log *logger.Logger) (*Client, error) {
 	action := action.NewInstall(actionConfig)
 	action.Namespace = constants.HelmNamespace
 	action.Timeout = timeout
-	action.Atomic = true
-	action.Wait = true
+	// TODO(malt3): Enable "Wait" once every service gets ready with only
+	// one control-plane node and no worker nodes.
+	action.Atomic = false
+	action.Wait = false
```

Comment:
does Atomic also have to be reverted?

Reply:
Yes. If you follow the usage of Wait into helms source code, you'll find that somewhere there is a statement like `Wait = Wait | Atomic`. I'm on mobile but will update this with a link to the source.